Instruction context:
   
 L. 336       220  LOAD_GLOBAL              IndexError
                 222  CALL_FUNCTION_0       0  '0 positional arguments'
                 224  RAISE_VARARGS_1       1  'exception instance'
                 226  JUMP_BACK            28  'to 28'
                 228  POP_BLOCK        
->             230_0  COME_FROM_LOOP       18  '18'

test/439.pyc -- 
# decompile failed

# file test/439.pyc
# Deparsing stopped due to parse error
